client/asset/eth/nodeclient.go

```diff
@@ -207,20 +202,32 @@ func (n *nodeClient) balance(ctx context.Context) (*Balance, error) {
 	}
 
 	outgoing := new(big.Int)
+	incoming := new(big.Int)
+	zero := new(big.Int)
 	for _, tx := range pendingTxs {
 		// TODO: Check that the from address matches addr. types.Transaction
 		// does not expose the from address directly.
```

Comment:
Think it would be this:

```go
ethSigner := types.LatestSigner(n.leth.ApiBackend.ChainConfig()) // "latest" good for pending
...
for _, tx := range pendingTxs {
	from, _ := ethSigner.Sender(tx) // zero Address on error
	if from != n.address() {
		continue
	}
```